From 6759eff8ebb6003b83c1bf13a45e386fe56c74ad Mon Sep 17 00:00:00 2001
From: Shenghou Ma <minux@golang.org>
Date: Mon, 7 Nov 2016 19:08:51 -0500
Subject: [PATCH 32/38] [release-branch.go1.7] doc/devel/release.html: document
 go1.6.3 doesn't actually support macOS Sierra

Updates #17824.

Change-Id: I73cf89c21b418158c7014c3271cd1103a17a5c86
Reviewed-on: https://go-review.googlesource.com/32882
Reviewed-by: Brad Fitzpatrick <bradfitz@golang.org>
Reviewed-on: https://go-review.googlesource.com/32885
Reviewed-by: Ian Lance Taylor <iant@golang.org>
Reviewed-by: Minux Ma <minux@golang.org>
---
 doc/devel/release.html | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/devel/release.html b/doc/devel/release.html
index 9a4daf1..eff0772 100644
--- a/doc/devel/release.html
+++ b/doc/devel/release.html
@@ -89,7 +89,7 @@ See the <a href="https://github.com/golang/go/issues?q=milestone%3AGo1.6.2">Go
 <p>
 go1.6.3 (released 2016/07/17) includes security fixes to the
 <code>net/http/cgi</code> package and <code>net/http</code> package when used in
-a CGI environment. This release also adds support for macOS Sierra.
+a CGI environment.
 See the <a href="https://github.com/golang/go/issues?q=milestone%3AGo1.6.3">Go
 1.6.3 milestone</a> on our issue tracker for details.
 </p>
-- 
2.7.4

